src/mase_components/deps.py

```diff
@@ -9,39 +9,51 @@
 
 MASE_HW_DEPS = {
     # Activations
-    "activations/fixed_relu": [],
-    "activations/fixed_leaky_relu": [],
-    "activations/fixed_tanh": ["cast"],
-    "activations/fixed_selu": ["cast", "activations", "fixed_math", "activations"],
-    "activations/fixed_gelu": ["fixed_arithmetic", "common", "activations"],
-    "activations/fixed_softsign": ["fixed_arithmetic", "common", "activations"],
-    "activations/fixed_softplus": ["activations"],
-    "activations/fixed_hardshrink": ["common", "cast"],
-    "activations/fixed_hardswish": ["common", "fixed_arithmetic"],
-    "activations/fixed_silu": ["common", "cast", "activations"],
-    "activations/fixed_elu": ["common", "cast", "activations"],
-    "activations/fixed_sigmoid": ["common", "cast", "activations"],
-    "activations/fixed_softshrink": ["common", "cast"],
-    "activations/fixed_logsigmoid": ["common", "cast", "activations"],
-    "activations/fixed_softmax": [
+    "activation_layers/fixed_relu": [],
+    "activation_layers/fixed_leaky_relu": [],
+    "activation_layers/fixed_tanh": ["cast"],
+    "activation_layers/fixed_selu": [
+        "cast",
+        "activation_layers",
+        "scalar_operators/fixed",
+    ],
+    "activation_layers/fixed_gelu": [
+        "common",
+        "memory",
+        "activation_layers",
+    ],
+    "activation_layers/fixed_softsign": [
+        "old",
```

Comment:
We need `fixed_mult` op here so we might need to move files in the `old` folder to the right folders... @Aaron-Zhao123

Reply:
Yep, I restructured things a bit again in `linear_layer`.